source/extensions/filters/udp/udp_proxy/udp_proxy_filter.cc

```diff
@@ -309,6 +309,11 @@ UdpProxyFilter::ActiveSession::~ActiveSession() {
       .connections()
       .dec();
 
+  // Clearing filters list to allow destructors of session filters to run
+  // and write to filter state before writing session access log.
+  write_filters_.clear();
+  read_filters_.clear();
```

Comment:
Does this need runtime guard and release note?

Reply:
Added, thanks!